RestaurantReservation.Db/Repositories/EmployeeRepository.cs

```diff
@@ -0,0 +1,22 @@
+﻿using Microsoft.EntityFrameworkCore;
+using RestaurantReservation.Db.Models;
+
+namespace RestaurantReservation.Db.Repositories
+{
+    public class EmployeeRepository
+    {
+        private readonly RestaurantReservationDbContext _context;
+
+        public EmployeeRepository(RestaurantReservationDbContext context)
+        {
+            _context = context;
+        }
+
+        public async Task<List<Employee>> ListManagersAsync()
+        {
+            return await _context.Employees
+                                 .Where(e => e.Position == "Manager")
```

Comment:
Lets save hard coded strings in a constant class, so they are reusable, and one place to change. `EmployeePositions`